code/datums/mutations/antenna.dm

```diff
@@ -62,6 +62,14 @@
 
 	ranged_mousepointer = 'icons/effects/mouse_pointers/mindswap_target.dmi'
 
+/datum/action/cooldown/spell/pointed/mindread/Grant(mob/grant_to)
+	. = ..()
+	grant_to.add_traits(TRAIT_MIND_READER, GENETIC_MUTATION)
```

Comment:
This will not work because add_traits expects a list, just use `ADD_TRAIT`

Reply:
Also I thiiink you need to check `if(!owner) return` for `Grant`, because it can fail in some contexts? Vague memory of that

Reply:
I can't find anything consistently about that, but I'll add it just in case.